pytest_plugins/issue_handlers.py

```diff
@@ -16,20 +16,44 @@
 from robottelo.utils.version import search_version_key
 from robottelo.utils.version import VersionEncoder
 
-LOGGER = logging.getLogger('issue_handlers_plugin')
+LOGGER = logging.getLogger('robottelo')
```

Comment:
There is no specific configuration in logging.yaml for a logger for this file, and I don't necessarily think it needs its own?

This results in the log messages I added in the PR being console and file printed during execution, after the pytest warnings, with the default logging config. I could understand not wanting this change to logging verbosity during collection, but it only logs when BZ filtering is active.